applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,81 @@
+.. _slm_as_zephyr_modem:
+
+nRF91 Series as a Zephyr-compatible modem
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+Starting with |NCS| v2.6.0, the Serial LTE Modem (SLM) application can be used to turn an nRF91 Series SiP into a standalone modem that can be used through Zephyr's generic modem driver.
+This means that the controlling chip can run a Zephyr application that seamlessly uses Zephyr's IP stack instead of offloaded sockets via AT commands.
+
+This is made possible by SLM's support of CMUX and PPP and Zephyr's generic modem driver.
+
+The :ref:`nRF52840 SoC present in the nRF9160 DK <zephyr:nrf9160dk_nrf52840>` can even be made to be the controlling chip with the appropriate configuration (described below).
```

Comment:
Should this mention that other nrf91 products do not support this?

Reply:
I'm actually still surprised about this. Tried to look at the documentation of the nRF9161 DK but it was pretty sparse (compared to that of the nRF9160 DK).